src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
```

Comment:
**Issue**: Initializing absMax to 0 can lead to incorrect results if all numbers in the array are negative. <br> **Fix**: Initialize absMax with the first element of the array instead of 0 to handle cases with all negative numbers correctly. <br> **Code Suggestion**: 
 ```
 -        int absMax = 0;
 +        int absMax = numbers[0];
 ```